test/WebSites/RazorWebSite/Components/ComponentWithRelativePath.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace RazorWebSite.Components
+{
+    public class ComponentWithRelativePath : ViewComponent
+    {
+        public IViewComponentResult Invoke(Person person)
+        {
+            return View("../Shared/Components/ComponentWithRelativePath.cshtml", person);
```

Comment:
Would we need tests to verify the behavior of relative paths outside app root? I'm guessing that's testing the implementation of `IFileProvider` at that point.